test/test_shape_ops.py

```diff
@@ -605,7 +606,19 @@ def test_nonzero_non_diff(self, device):
         nz = x.nonzero()
         self.assertFalse(nz.requires_grad)
 
+class TestShapeFuncs(TestCase):
```

Comment:
Add a comment here describing what this class is for. 

Do we really want to add another test class instead of just putting this function into the existing TestShapeOps?

Reply:
I just thought, going forward all `ShapeFuncsInfo` Op tests might as well go under `TestShapeFuncs`. But we can just move it to the existing class as well.

Do let me know if that is preferred.

Thanks!

Reply:
Either way is fine, just add a comment explaining why people should put a test here, for example, vs. the other test suite.